test/integration/admin/api/api_docs_services_controller_test.rb

```diff
@@ -26,4 +26,88 @@ def current_account
       master_account
     end
   end
+
+  class ProviderAccountTest < Admin::Api::ApiDocsServicesControllerTest
+    setup do
+      @provider = FactoryGirl.create(:provider_account)
```

Comment:
Can it be `:simple_provider` instead?
Then create a `:simple_service`

Reply:
Ok 👍 If i have to push for anything else bigger than this, I will push it too, otherwise i will do this change in the next PR